test/functional/tests/security/fuzzy/config/io_class_allocation.yml

```diff
@@ -0,0 +1,14 @@
+- name: String
+  attributes:
+    name: Bit
```

Comment:
Name "Bit" here is a bit misleading, as this value is in fact fixed point number.